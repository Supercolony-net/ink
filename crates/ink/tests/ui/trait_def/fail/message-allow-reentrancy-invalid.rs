#[ink::trait_definition]
pub trait TraitDefinition {
    #[ink(message, allow_reentrancy = false)]
    fn message(&self);
}

fn main() {}
