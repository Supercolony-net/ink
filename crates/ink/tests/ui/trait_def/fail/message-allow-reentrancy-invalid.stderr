error: unknown ink! attribute argument (name = value)
 --> tests/ui/trait_def/fail/message-allow-reentrancy-invalid.rs:3:20
  |
3 |     #[ink(message, allow_reentrancy = false)]
  |                    ^^^^^^^^^^^^^^^^^^^^^^^^
